stubs/pyOpenSSL/OpenSSL/crypto.pyi

```diff
@@ -98,7 +98,7 @@ class X509Req:
     def to_cryptography(self) -> CertificateSigningRequest: ...
     def verify(self, pkey: PKey) -> bool: ...
 
-# deprecated
+@deprecated("X509Extension support in pyOpenSSL is deprecated. You should use the APIs in cryptography.")
```

Comment:
Where are you seeing that this is deprecated? I didn't find any mention of that in the source or at https://www.pyopenssl.org/en/latest/api/crypto.html#x509extension-objects.

The docs have a comment saying people should preferably use cryptography instead, but I don't think that should be considered a deprecation.